### PR TITLE
pythonPackages.pam: init at 1.8.4

### PR DESCRIPTION
**Pull request by mkg20001**
###### Motivation for this change

Needed for cinnamon

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/pam/default.nix
Comment (cole-h) [inline pkgs/development/python-modules/pam/default.nix:28]:
```suggestion
    homepage = "https://github.com/FirefighterBlu3/python-pam";
```

File: pkgs/development/python-modules/pam/default.nix
Comment (cole-h) [inline pkgs/development/python-modules/pam/default.nix:30]: This needs a maintainer.

File: pkgs/top-level/python-packages.nix
Comment (cole-h) [inline pkgs/top-level/python-packages.nix:3101]: Could also be written as:

```suggestion
    inherit (pkgs) pam;
```

**Review by cole-h — CHANGES_REQUESTED** [review]

**Review by jonringer — APPROVED** [review]
diff LGTM after @cole-h 's suggestions

**Review by cole-h — APPROVED** [review]
Diff LGTM. Builds fine.

```
[5 built, 33 copied (89.0 MiB), 16.0 MiB DL]
https://github.com/NixOS/nixpkgs/pull/84544
25 package marked as broken and skipped:
gdata-sharp iterm2 linuxPackages_hardkernel_4_14.bcc linuxPackages_hardkernel_4_14.can-isotp linuxPackages_hardkernel_4_14.chipsec linuxPackages_hardkernel_4_14.digimend linuxPackages_hardkernel_4_14.evdi linuxPackages_hardkernel_4_14.mba6x_bl linuxPackages_hardkernel_4_14.nvidia_x11 linuxPackages_hardkernel_4_14.nvidia_x11_legacy390 linuxPackages_hardkernel_4_14.nvidiabl linuxPackages_hardkernel_4_14.prl-tools linuxPackages_hardkernel_4_14.rtl8821ce nix-linter octave-jit octoprint php74Extensions.couchbase php74Extensions.pthreads python27Packages.caffe python27Packages.habanero python27Packages.handout python27Packages.hass-nabucasa python37Packages.pyblock python38Packages.pyblock synfigstudio

3 package built:
python27Packages.pam python37Packages.pam python38Packages.pam
```

**Review by jonringer — APPROVED** [review]
diff LGTM

```
https://github.com/NixOS/nixpkgs/pull/84544
25 package marked as broken and skipped:
gdata-sharp iterm2 linuxPackages_hardkernel_4_14.bcc linuxPackages_hardkernel_4_14.can-isotp linuxPackages_hardkernel_4_14.chipsec linuxPackages_hardkernel_4_14.digimend linuxPackages_hardkernel_4_14.evdi linuxPackages_hardkernel_4_14.mba6x_bl linuxPackages_hardkernel_4_14.nvidia_x11 linuxPackages_hardkernel_4_14.nvidia_x11_legacy390 linuxPackages_hardkernel_4_14.nvidiabl linuxPackages_hardkernel_4_14.prl-tools linuxPackages_hardkernel_4_14.rtl8821ce nix-linter octave-jit octoprint php74Extensions.couchbase php74Extensions.pthreads python27Packages.caffe python27Packages.habanero python27Packages.handout python27Packages.hass-nabucasa python37Packages.pyblock python38Packages.pyblock synfigstudio

3 package built:
python27Packages.pam python37Packages.pam python38Packages.pam
```

**Pull request merged**
